variables.tf

```diff
@@ -9,6 +9,12 @@ variable "lacework_agent_name" {
   default     = "lacework-agent"
 }
 
+variable "lacework_agent_interface_connection_size" {
+  type        = string
+  description = "Desired value for the InterfaceConnectionSize Lacework agent parameter"
+  default     = "10000"
```

Comment:
I wonder if these are the default that the agent sets and if not, I wonder what are the implications of setting this by default. I will check with the Agent Team internally.